packages/@o3r/design/src/core/design-token/parsers/design-token.parser.spec.ts

```diff
@@ -96,6 +96,50 @@ describe('Design Token Parser', () => {
         expect(item).toBeDefined();
         expect(item.extensions.o3rImportant).toBe(false);
       });
+
+      test('should generate a variable with template when token matching star', () => {
+        const result = parser.parseDesignToken({
+          ...exampleVariableWithContext,
+          context: {
+            template: {
+              example: {
+                // eslint-disable-next-line @typescript-eslint/naming-convention
+                '*': {
+                  $extensions: {
+                    o3rImportant: true
+                  }
+                }
+              }
+            } as DesignTokenGroupTemplate
+          }
+        });
+        const item = result.get('example.var1');
+
+        expect(item).toBeDefined();
+        expect(item.extensions.o3rImportant).toBe(true);
+        expect(item.extensions.o3rPrivate).toBeFalsy();
+      });
+
+      test('should generate a variable with template when token path matching star', () => {
```

Comment:
both tests do the same thing? should delete duplicate or update the content of the second test

Reply:
duplicated test, I removed one